packages/terra-site/themeable-variables.json

```diff
@@ -1,5 +1,5 @@
 {
-  "C:\\workspace\\dv045327_personal_repos\\terra-core\\packages\\terra-alert\\lib\\Alert": {
```

Comment:
Looks like you already got it 😄

Reply:
Yep! Definite "oh no" moment! Just curious, seems like these full paths are auto generated on Windows but don't seem to be an issue for others, are there any issues logged to terra-core about this? I haven't done too much investigation on what this file does or what causes this path to get here.

Reply:
I'm not sure if there is an issue logged, but this behavior is something we are aware of. Our themeable-variables plugin needs updated to handle windows file paths. If I remember its checking for '/' to split the path and since windows is '\\' it returns the full thing.